examples/docker-compose/notebook/up.sh

```diff
@@ -6,12 +6,12 @@ set -e
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-USAGE="Usage: `basename $0` [--secure | --letsencrypt] [--password PASSWORD] [--secrets SECRETS_VOLUME]"
+USAGE="Usage: $(basename "$0") [--secure | --letsencrypt] [--password PASSWORD] [--secrets SECRETS_VOLUME]"
```

Comment:
quotes within quotes probably causing an issue here.

Reply:
It seems to be ok to use quotes in quotes for command substitution in a new style.
https://stackoverflow.com/questions/17984958/what-does-it-mean-in-shell-when-we-put-a-command-inside-dollar-sign-and-parenthe

Reply:
Ah thanks for clarifying this